src/content/datachannel/js/main.js

```diff
@@ -67,13 +71,15 @@ function createConnection() {
     }
   }
   // Add localConnection to global scope to make it visible from the browser console.
-  window.localConnection = localConnection = new RTCPeerConnection(servers, pcConstraint);
+  window.localConnection = localConnection =
+    new RTCPeerConnection(servers, pcConstraint);
```

Comment:
all wrapped lines should indent 4. I mentioned several below, but there are many others.